.env

```diff
@@ -26,6 +29,10 @@ FCREPO_LOG_LEVEL=INFO
 FCREPO_LOG_JMS=INFO
 FCREPO_ACTIVEMQ_CONFIGURATION=classpath:/activemq-queue.xml
 
+COMPACTION_URI=http://example.org/pass/
```

Comment:
The compaction URI should be `https://oa-pass.github.io/pass-data-model/src/main/resources/context-2.0.jsonld`, as that is the URI of the context used for compaction.  It is built in to the fcrepo image by default.  Ember should be using that as `@context` too.

Reply:
I have two docker setups that have been doing it this way with success for a while so it would be good to clarify. Is this wrong and it just works be chance?

When ember constructs JSON-LD, it is using `@context: http://example.org/pass/`
Isn't that what we want? Since Fedora has builtin knowledge of what context the http://example.org/pass/ resolves too, it seems we don't have to use the github URI.

Reply:
Right.  So the `CONTEXT_*` stuff is used to bake in a context value so that Fedora can resolve it without using the internet.

The issue is that all the backend services are using `https://oa-pass.github.io/pass-data-model/src/main/resources/context-2.0.jsonld` as the context URI.  This is what is baked into the image.  The Ember config is forcing it to be `http://example.org/pass/` instead.. which works.  It is not by chance that `pass-ember` is working.  It is working because it forced the context URI to be something unique to that environment, using the tools provided to allow that. 

However, we really want:
1. Everybody to use the same context URIs 
2. The context URIs to be resolvable by anybody in the world who might encounter the jsonld and want to interpret it as rdf, as it is expected by the jsonld spec

Using `https://oa-pass.github.io/pass-data-model/src/main/resources/context-2.0.jsonld` satisfies both conditions.  

Note that once we have `oapass.org` up and running, we'll change that URI to something more palitable, like `http://oapass.org/ns/context-2.0.jsonld`, or something along those lines.  'till then, we have github.

Reply:
Ok. Good. It sounds like I just need to change it for this project. I'll give it a shot.